components/selectors/parser.rs

```diff
@@ -103,6 +103,11 @@ macro_rules! with_all_bounds {
 
             /// pseudo-elements
             type PseudoElement: $($CommonBounds)* + PseudoElement<Impl = Self>;
+
+            /// Returns whether the selector matches conditions for the :active and
+            /// :hover quirk.
```

Comment:
Also, this comment is false, this only returns whether the pseudo-class is `:hover` or `:after`, right?